src/Components/LabSwap/DisplayBlock.tsx

```diff
@@ -29,64 +30,6 @@ export const DisplayBlock: FC<Props> = ({visible, size, inline, data}) => {
     return `${hour12}:${minutes} ${ampm}`;
   }
 
-  const renderScheduled = (retData: CourseBlock[]) => {    
-    let retFormat: DisplayBlock[] = [];
-    let ret: JSX.Element[] = [];
-
-    const shortDays = ['M', 'T', 'W', 'R', 'F'];
-    const dayMap = new Map<string, shortday>(
-      [
-        ['Monday', 'M'],
-        ['Tuesday', 'T'],
-        ['Wednesday', 'W'],
-        ['Thursday', 'R'],
-        ['Friday', 'F']
-      ]
-    )
-
-    const cmpDay = (a: shortday, b: shortday) => {
-      if (shortDays.indexOf(a) < shortDays.indexOf(b)) return -1;
-      else if (shortDays.indexOf(a) > shortDays.indexOf(b)) return 1;
-      return 0;
-    }
-
-    retData.sort((a, b) => {
-      if (cmpDay(dayMap.get(a.weekday)!, dayMap.get(b.weekday)!) !== 0) return cmpDay(dayMap.get(a.weekday)!, dayMap.get(b.weekday)!);
-      else if (a.start_time < b.start_time) return -1;
-      else if (a.start_time > b.start_time) return 1;
-      else if (a.course_number < b.course_number) return -1;
-      else if (a.course_number > b.course_number) return 1;
-      else if (a.section_number < b.section_number) return -1;
-      else if (a.section_number > b.section_number) return 1;
-      return 0;
-    }).forEach(block => {
-      const where = retFormat.findIndex(b => b.course_number === block.course_number && b.section_number === block.section_number);
-      if (where === -1) retFormat.push({...block, days: [dayMap.get(block.weekday) as shortday]})
-      else retFormat[where].days.push(dayMap.get(block.weekday) as shortday);
-    })
-
-    retFormat.forEach(block => {
-      ret.push(
-        <div className="schedule-info-element" key={`sie-${JSON.stringify(block)}`}>
-          <div>
-            {block.department}: {block.course_number}-{block.section_number}
-          </div>
-          <div>
-            {formatDate(block.start_time)}-{formatDate(block.end_time)} {block.days.join('')}
-          </div>
-          <div>
-            Professor {block.professor}
-          </div>
-          <div>
-            {block.place}
-          </div>
-        </div>
-      )
-    })
-
-    return ret;
-  }
```

Comment:
I do not know why this function was here, it wasn't being called so I took it out